yoti-sdk.podspec

```diff
@@ -16,7 +16,7 @@ Pod::Spec.new do |s|
   #
 
   s.name         = "yoti-sdk"
-  s.version      = "1.1"
+  s.version      = "2.1"
```

Comment:
Why is the version 2.1 as opposed to 2.0 ?

Reply:
We already have 2.0 in last December and apparently we forgot to update the podspec at that time. So this release will be 2.1